spring/src/test/java/io/cucumber/spring/threading/RunParallelCucumberTest.java

```diff
@@ -1,21 +1,23 @@
 package io.cucumber.spring.threading;
 
 import io.cucumber.core.cli.Main;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.RepeatedTest;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Future;
 
 import static java.util.concurrent.Executors.newFixedThreadPool;
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.jupiter.api.Assertions.assertAll;
 
-class RunParallelCukesTest {
+class RunParallelCucumberTest {
 
-    @Test
-    void test() {
+    @RepeatedTest(100)
```

Comment:
We're running 100 repetitions with 100 concurrent scenarios. Should tone this down a bit for CI but good enough.

Reply:
```
[INFO] Cucumber-JVM: Spring ............................... SUCCESS [02:12 min]
```
:rofl: